server/api.js

```diff
@@ -35,5 +37,8 @@ apiRouter.use('/lob', lob)
 apiRouter.use('/checkout', checkout)
 //apiRouter.use('/twilio', twilio)
 apiRouter.use('/event_logger', eventLogger)
+// ADD V1 AND ADMIN ROUTES
```

Comment:
This will work, but it will create two url paths: `api/v1/admin` and `api/admin`. We only want to be able to modify admins under the v1 schema, so the other is unncessary.